src/app.ts

```diff
@@ -33,7 +33,8 @@ async function main() {
       const manager = getManager();
       // find the user who has a token in their sessions field
       const session = await manager.findOne(UserSessionModel, { accessToken: accessToken });
-      if (session) {
+      // check if the session token has expired
+      if (session && session.expiresAt.getTime() > Date.now()) {
```

Comment:
i'm not sure how this works for all users but assuming it does we should be doing this session token expiration test not only when the express server is created but also when the token itself expires (not sure where that is done in the code)

Reply:
it does haha